README.md

```diff
@@ -9,12 +9,27 @@
 	<br>
 </h1>
 
-> Repo for Filecoin Station Core
+> Station Core is a headless version of Filecoin Station suitable for running on
+> all kinds of servers.
 
 [![CI](https://github.com/filecoin-station/core/actions/workflows/ci.yml/badge.svg)](https://github.com/filecoin-station/core/actions/workflows/ci.yml)
 
+## Deployment
+
+Station Core supports different deployment options:
+
+- [Docker](#docker)
+- [Manual Deployment (Ubuntu)](#manual-deployment-ubuntu)
+
 ## Installation
 
+> **Note**: Station Core requires Node.js, we recommend using the latest LTS
+> version. You can install Node.js using your favorite package manager or get
+> the official installer from
+> [Node.js downloads](https://nodejs.org/en/download/).
```

Comment:
Rendered version:

<img width="1045" alt="Screenshot 2023-03-15 at 14 36 13" src="https://user-images.githubusercontent.com/1140553/225325014-82d41a01-f9f1-4db2-8f22-9221cd75b772.png">

See https://github.com/community/community/discussions/16925

Reply:
This is looking awesome! 👏